src/client/pages/_slug.vue

```diff
@@ -62,39 +31,21 @@ export default {
         socialTarget: this.url
       })
     }
-  }
-}
-</script>
-
-<style>
-@import '../components/app-core/index.css';
-
-.page__content {
-  margin: 0 auto;
-  padding: 0 var(--spacing-default);
-  max-width: 920px;
-}
+  },
 
-.page__sidebar {
-  margin: 0 auto var(--spacing-double) auto;
-  width: 100%;
-  max-width: var(--max-width);
-}
+  computed: {
+    shareText() { return this.page.seo.description },
+    shareTitle() { return this.page.seo.title },
+  },
 
-@media screen and (min-width: 880px) {
-  .page__content {
-    display: flex;
-    justify-content: space-between;
-    align-items: flex-start;
-    padding: 0 var(--spacing-double);
-  }
+  data () {
+    return {
+      url: '',
+    }
+  },
 
-  .page__sidebar {
-    position: sticky;
-    top: calc(var(--app-header-height) + var(--spacing-double));
-    flex-grow: 1;
-    margin: 0 var(--spacing-double) 0 0;
-    max-width: 300px;
+  mounted () {
+    this.url = window.location.href
```

Comment:
so this is not necessary anymore right?

Reply:
Yes, but this change is a requirement on the Enhanced sharing button, so in order to not mix things up and since it's related to it, I'll fix it when working on that ticket. 😄